### ISSUE TITLE
"tenant_command" + "dumpdata" = doesn't work?

### ISSUE THREAD

**Issue by bartmika** [tomturner/django-tenants#95]
Hi,

I would like to export data in a tenant schema to a json fixture using ``dumpdata`` command which comes with **Django**.

I am running the following code:

```bash
python manage.py tenant_command dumpdata --indent 4 --format=json foundation_tenant.NAICSOption > ~/Desktop/niacsoption.json
```

And I get the message:

```bash
manage.py tenant_command: error: unrecognized arguments: --indent 4 --format=json foundation_tenant.NAICSOption
```

Does anyone know what I did wrong, or is this a problem with ``tenant_command``?

**Comment by rodolfomartinez** [comment]
+1

**Comment by tomturner** [comment]
Fixed IN 1.1.10 however the command is a bit diffent

> python manage.py tenant_command dumpdata --indent 4 --format=json --s foundation_tenant.NAICSOption > ~/Desktop/niacsoption.json
